crates/burn-import/src/burn/node/base.rs

```diff
@@ -13,6 +13,7 @@ use super::{
     reshape::ReshapeNode, resize::ResizeNode, slice::SliceNode, squeeze::SqueezeNode, sum::SumNode,
     unary::UnaryNode, unsqueeze::UnsqueezeNode,
 };
+use crate::burn::node::mean::MeanNode;
```

Comment:
Minor nitpick: since all other nodes are already imported with `use super::`, let's add `super::mean::MeanNode` to the list 🙂